### PR TITLE
Don't over praise when questions are incorrect

### PR DESCRIPTION
**Pull request by nathanstitt**
This shows different messages depending on how many questions are correct for readings and practices.

all correct: Great job, all questions answered correctly!
1 incorrect: Good job, only 1 question incorrect!
anything else:  <#incorrect> questions were incorrect, try harder next time

screen shot 2015-05-27 at 2 51 01 pm

### CODE REVIEW

**Comment by nathanstitt** [comment]
@Fredasaurus thoughts on this wording?

**Comment by Fredasaurus** [comment]
I like the idea, let's work on the wordsmithing before demo.

Telling a student they just need to try harder when they don't get something may not be the best motivation technique. I'll run it past Jason/Kathi.

**Comment by pandafulmanda** [comment]
hahaha funny

File: src/flux/task.coffee [hunk 1]
```diff
@@ -117,6 +117,15 @@ TaskConfig =
...
       steps = getCompleteSteps(allSteps)
 
+    # return a feedback message with varying amounts of praise for the number of questions correct.
+    # Used by practice and readings.  The number of questions correct is not shown for homeworks
+    getFeedbackMessage: (taskId) ->
+      incorrect = _.where(this._steps[taskId], is_correct: false).length
+      switch incorrect
+        when 0 then "Great job, all questions answered correctly!"
+        when 1 then "Good job, only 1 question incorrect!"
+        else "#{incorrect} questions were incorrect, try harder next time"
```
Comment (philschatz) [inline src/flux/task.coffee]: eep, should it really be "try harder next time"?

Comment (nathanstitt) [inline src/flux/task.coffee]: No, almost certainly not - that's really just a placeholder until Fred gets proper verbiage.

**Comment by jpslav** [comment]
Great PR name.

**Comment by gregfitch** [comment]
I think this whole concept is putting judgment evaluations precisely where we DON'T want them.  For readings, the Finish page should read: "You are done. Great job!"  The great job is on completing the reading, not performance on concept questions.  For Practices, until we get a better motivation/gamification in place, the Finish page should read, "You're done with this round of questions.  Keep practicing to improve your understanding."

**Comment by gregfitch** [comment]
Sorry, somehow I'm in GitHub at Greg, but the comment above is from me, Jason Holmes.

**Pull request merged**